aten/src/ATen/native/EmbeddingBag.cpp

```diff
@@ -237,6 +235,62 @@ Tensor _embedding_bag_backward(const Tensor &grad, const Tensor &indices,
   }
 }
 
+static Tensor _embedding_bag_dense_backward_cpu_max(
```

Comment:
I'm assuming this is just a copy-paste of a block of code previously part of _embedding_bag_dense_backward_cpu.

Reply:
yes